verifier/verifier.go

```diff
@@ -257,6 +284,15 @@ func InitVerifier(config *configModel.Configuration) (err error) {
 		logging.Log().Errorf("Was not able to initiate a signing key. Err: %v", err)
 		return err
 	}
+
+	didSigningKey, err := getRequestSigningKey(verifierConfig.ClientIdentification.KeyPath, verifierConfig.ClientIdentification.Id)
+	if (slices.Contains(verifierConfig.SupportedModes, REQUEST_MODE_BY_VALUE) || slices.Contains(verifierConfig.SupportedModes, REQUEST_MODE_BY_REFERENCE)) && err != nil {
+		logging.Log().Errorf("Was not able to get a signing key, despite mode %s supported. Err: %v", REQUEST_MODE_BY_VALUE, err)
+		return err
+	} else {
+		err = nil
+	}
+
 	logging.Log().Warnf("Initiated key %s.", logging.PrettyPrintObject(key))
```

Comment:
can you move this line over the new code so its with the init of the field "key"?

Reply:
More left overs